config.yaml

```diff
@@ -5,6 +5,8 @@ common:
  log_size: 2097152
  packages_metadata_repo_url: "https://github.com/open-power-host-os/versions.git"
  packages_metadata_repo_branch: 'master'
+ packages_metadata_repo_ref_specs:
```

Comment:
I'd use `refspec` in one word since that is what git uses. It's also easier on people not acquainted with the term. Two words raises the question if/which of them are abbreviations (reference specification? referee's spectacles?)